src/MainWindow.vala

```diff
@@ -44,15 +46,19 @@ public class Tasks.MainWindow : Hdy.ApplicationWindow {
     static construct {
         Hdy.init ();
 
+        action_accelerators[ACTION_ADD_NEW_LIST] = "<Control>N";
```

Comment:
Do you really think we need an accelerator for creating a new list? IMHO this is quite a rare action - and I'd think if we use this shortcut, it would be more usefull for adding a new task?

Reply:
Put some more thoughts into this, the `<Control>N` shortcut may come in handy for: https://github.com/elementary/tasks/issues/148. Therefore resolving and merging this 🥳